api/src/main/resources/application.conf

```diff
@@ -20,6 +20,12 @@ akka {
   cluster {
     failure-detector.threshold = 12 //Increase value for AWS environments
     metrics.enabled = off
+
+    retry-unsuccessful-join-after = 20s
+
+    //DON'T USE IN PRODUCTION!!!!!
```

Comment:
Okay. Let's make sure there's an open issue in GH to remind us to update that value before we go to production.